src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,43 +1,81 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+  isLoading: boolean;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose, isLoading }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoadingUser, setIsLoadingUser] = useState(true);
+
+  useEffect(() => {
+    setIsLoadingUser(true);
+
+    getUser(todo.userId)
+      .then(userFromServer => {
+        setUser(userFromServer);
+      })
+      .finally(() => {
+        setIsLoadingUser(false);
+      });
+  }, [todo.userId]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      <div className="modal-card">
+        {isLoading || isLoadingUser ? (
```

Comment:
```suggestion
        {isLoadingUser ? (
```
looks redundant for me